doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,151 +23,309 @@ For testing purposes only. For testing purposes only. For testing purposes only.
+
+   bucket_name       = "my-test-bucket"
+   db_identifier     = "my-test-db"
+   database_password = "enter-your-secure-password"
+
+   tags = {
+     Environment = "my-test"
+     Team        = "my-test-team"
+     Project     = "my-test-project"
+   }
+
+   node_group_ami_type       = "AL2023_ARM_64_STANDARD"
+   node_group_instance_types = ["r6g.medium"]
+   node_group_desired_size   = 3
+   node_group_min_size       = 2
+   node_group_max_size       = 5
+
+   db_instance_class    = "db.t3.large"
+   db_allocated_storage = 20
+   db_multi_az          = false
+
+   enable_cluster_creator_admin_permissions = true
+   ```
+
+1. Initialize the terraform directory.
+
+    ```bash
+    terraform init
+    ```
+
+1. Create a terraform plan and review the changes.
+
+    ```bash
+    terraform plan -out my-plan.tfplan
+    ```
 
-1. If installing for the first time, create a namespace. The default
-   configuration uses the `materialize` namespace.
+1. If you are satisfied with the changes, apply the terraform plan.
+
+    ```bash
+    terraform apply my-plan.tfplan
+    ```
+
+   <a name="terraform-output"></a>
+   Upon successful completion, various fields and their values are output:
 
    ```bash
-   kubectl create namespace materialize
+   ...
+
+   Apply complete! Resources: 71 added, 0 changed, 0 destroyed.
+
+   Outputs:
+   database_endpoint = "my-test-db.abcdefg8dsto.us-east-1.rds.amazonaws.com:5432"
+   eks_cluster_endpoint = "https://0123456789A00BCD000E11BE12345A01.gr7.us-east-1.eks.amazonaws.com"
+   materialize_s3_role_arn = "arn:aws:iam::000111222333:role/my-test-materialize-s3-role"
+   metadata_backend_url = <sensitive>
+   oidc_provider_arn = "arn:aws:iam::000111222333:oidc-provider/oidc.eks. us-east-1.amazonaws.com/id/0123456789A00BCD000E11BE12345A01"
+   persist_backend_url = "s3://my-test-bucket/ my-test:serviceaccount:materialize-environment:12345678-1234-1234-1234-1234 56789012"
+   s3_bucket_name = "my-test-bucket"
+   vpc_id = "vpc-0abc000bed1d111bd"
    ```
 
-1. Create a `my-AWS-values.yaml` configuration file for the Materialize
-   operator. Update with details from your AWS Kubernetes environment. For more
-   information on cloud provider configuration, see the [Materialize Operator
-   Configuration](/self-managed/configuration/#operator-parameters).
+1. Note your specific values for the following fields:
+
+   - `materialize_s3_role_arn` (Used during [B. Install the Materialize
+     Operator](#b-install-the-materialize-operator))
+
+   - `database_endpoint` (Used during [C. Install
+     Materialize](#c-install-materialize))
+
+   - `persist_backend_url` (Used during [C. Install
+     Materialize](#c-install-materialize))
+
+
+1. If you do not have `kubectl` installed,
+
+   - Install `kubectl`. See the [Amazon EKS: install `kubectl`
+     documentation](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html).
+
+   - Configure `kubectl` to  connect to your EKS cluster, replacing:
```

Comment:
Might pull the config out to a separate step to run after people create their environment.